openedx/features/wikimedia_features/meta_translations/models.py

```diff
@@ -137,7 +137,7 @@ class WikiTranslation(models.Model):
     """
     Store translations fetched from wiki learn meta, will also manage mapping of source and target blocks.
     """
-    target_block = models.ForeignKey(CourseBlock, on_delete=models.CASCADE)
+    target_block = models.ForeignKey(CourseBlock, on_delete=models.CASCADE, related_name="course_block")
```

Comment:
why do we need this ?

Reply:
There are multiple ForeginKeys in the Course Block, that's why the _set command was not working in a serializer. To get wiki_translations from a block we need to name it.
The default name was not working i.e wiki_translation_set